src/config_objects.c

```diff
@@ -46,6 +46,13 @@ const struct NamedCommand objects_object_commands[] = {
   {"GENRE",           2},
   {"RELATEDCREATURE", 3},
   {"PROPERTIES",      4},
+  {"ANIMATIONID",     5},
+  {"ANIMATIONSPEED",  6},
+  {"SIZE_XY",         7},
+  {"SIZE_YZ",         8},
+  {"MAXIMUMSIZE",     9},
```

Comment:
I don't like that way to describe properties.
I think best way would be 
`{ <name>, <offset>, <type> }`
So any property would be
`((short*)objdat)[offset] = n`
instead of 
`objdat->size_xy = n;`

Reply:
After discussion on discord, such a change sounds more like refactoring all the code that loads in config files. That could go in a new commit/pull.